projects/igniteui-angular/src/lib/grids/cell.component.html

```diff
@@ -7,8 +7,8 @@
         [row]="rowData" [column]="this.column.field" [containerClass]="'igx-grid__td-text'" [metadata]="searchMetadata"
         class="igx-grid__td-text">{{ formatter ? formatter(value) : column.dataType === 'number' ? (value | igxdecimal:
         grid.locale) : column.dataType === 'date' ? (value | igxdate: grid.locale) : column.dataType === 'boolean' ? "" : value }}
-        <igx-icon *ngIf="column.dataType === 'boolean'" [ngClass]="{'igx-grid__icon--checked': value}" >{{value ? 'check' : 'close'}}</igx-icon>
     </div>
+    <igx-icon *ngIf="column.dataType === 'boolean'" [ngClass]="{'igx-grid__icon--checked': value}" >{{value ? 'check' : 'close'}}</igx-icon>
```

Comment:
```html
[ngClass]="{'igx-grid__icon--checked': value}
```
This should not be applied on the icon. The class should be different and applied on the `igx-grid-cell` directly.